packages/app/src/sandbox/compile.ts

```diff
@@ -707,20 +766,20 @@ async function compile(opts: CompileOptions) {
           process.env.LOCAL_SERVER ||
           process.env.SANDPACK
         ) {
-          // The HTML is loaded from the server as a static file, no need to set the innerHTML of the body
-          // on the first run. However, if there's no server to provide the static file (in the case of a local server
-          // or sandpack), then do it anyways.
-          document.body.innerHTML = body;
-
           // Add head tags or anything that comes from the template
```

Comment:
I think this comment is outdated, because this function doesn't override anything from the original head of the bundler page.